src/components/common/ChartContainer.tsx

```diff
@@ -10,7 +10,11 @@ const ChartContainer = ({
   children: JSX.Element;
 }): JSX.Element => (
   <Box width="100%">
-    <ResponsiveContainer width="100%" height={CONTAINER_HEIGHT}>
+    <ResponsiveContainer
+      width="100%"
+      height={CONTAINER_HEIGHT}
+      style={{ direction: 'ltr' }}
```

Comment:
Why is it set to `ltr`?

Reply:
I added this because in the case of right to left the x-axis and points are very close to the point you can't read ticks names, to fix this was the solution